feat: Add ability to disable Scheduled Job (#3447)
aws/copilot-cli: site/content/docs/manifest/scheduled-job.en.md

* `"* * * * *"` based on the standard [cron format](https://en.wikipedia.org/wiki/Cron#Overview).
* `"cron({fields})"` based on CloudWatch's [cron expressions](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#CronExpressions) with six fields.

> Inorder to disable the Scheduled Job you can set the `schedule` field to the value `none` for example:
> 
> ```yaml
> on:
>   schedule: "none"
> ```
> 
> This will disable the Eventbridge Rule 

Reviewer: ```suggestion
Finally, you can disable the job from triggering by setting the `schedule` field to `none` for example:
```yaml
on:
  schedule: "none"
```
```
